### PR TITLE
fix sidebar behaviour

### PR DESCRIPTION
**Pull request by gko**
As I said in my [previous PR](/nextcloud/notes/pull/422) there are couple of things that are bugging me, and I'll try to fix them.

Here I added closing behaviour when you change the notes.

## Motivation

Today when you change note on mobile phone the side bar actually stays, and it gets in the way of editing the note, as you need to  close it manually by clicking on `#app-navigation-toggle`.

I also got rid of some TODOs related to jQuery.

I replaced `animate` with a simple `scrollTop` animation although I think it might not be necessary at all.

👋 @korelstar

### CODE REVIEW

**Comment by korelstar** [comment]
Hi @gko,

thanks for your contribution, again!

These `closeNavbar` and `openNavbar` functions feel a little like a hack to me. I know, it wasn't better before, so we could use it a interim solution. However, it would be better, if the functionality to open/close the app-navigation would be provided by `@nextcloud/vue`. For one aspect of this, I already opened an issue over there: https://github.com/nextcloud/nextcloud-vue/issues/433. But a more general solution (i.e. the app-navigation can be manually opened and closes by the app; ideally independent of mobile mode) would be better. I opened a new issue for that: https://github.com/nextcloud/nextcloud-vue/issues/791. Please feel free to contribute to this, too :smiley: .

> I replaced `animate` with a simple `scrollTop` animation although I think it might not be necessary at all.

I agree, that animation isn't necessary. I'm fine with replacing the animation by a simple scroll to top without any animation. Could you change that?

File: src/App.vue [hunk 1]
```diff
@@ -192,8 +192,8 @@ export default {
 
 		onSelectCategory(category) {
 			this.filter.category = category
-			// TODO move the following from jQuery to plain JS
-			$('#app-navigation > ul').animate({ scrollTop: 0 }, 'fast')
+
+			scrollTop(document.querySelector('#app-navigation > ul'), 10000)
```
[the remark this answers is not in the document]
Comment (korelstar) [reply src/App.vue]: this should be replaced with a simple scroll to top without animation

**Comment by gko** [comment]
done. I'm working on fixing the rest of TODOs, and some other features, more PRs to follow.

**Comment by jancborchardt** [comment]
By the way @gko we now have a "Notes team" chat channel on our Nextcloud Talk instance. We can invite you with a guest account if you like? :)

**Pull request merged**